src/v2/auth.rs

```diff
@@ -337,7 +315,7 @@ mod tests {
         let header_value = HeaderValue::from_str(&format!(
             r#"Bearer realm="{}",service="{}",scope="{}""#,
             realm, service, scope
-        ))?;
+        )).expect("this statically known header value only contains ASCII chars so it is correct header value");
```

Comment:
The variables _realm_, _service_, and _scope_ aren't statically known.
Also, my rule of thumb is that if it's a 3rd party library I don't rely on it and avoid panicing.

Reply:
Bump.

Reply:
Ah, I see now that they are statically known here by expanding the previous lines. Also as you noted in your comment, this is in a test so panicking is appropriate.